python/postprocessing/wmass/postproc.py

```diff
@@ -178,10 +215,14 @@ class bcolors:
 else:
     kd_file += "_Data"
 kd_file += ".txt"
+if isTest:
+    kd_file = "keep_and_drop_TEST.txt"
```

Comment:
Hi @cippy I don't see this file added as a part of the PR.

Reply:
Added now